### PR TITLE
Guzzle options don't work if additionalConfigKeys not empty, after union

### PR DESCRIPTION
**Pull request by FI-LIFE**
Fixes #

## Changes proposed in this pull request:
- Replace union through "+"  to array_merge

### CODE REVIEW

File: src/Helpers/ConfigRetriever.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function fromServices($providerName, array $additionalConfigKeys = [])
         $this->providerName = $providerName;
         $this->getConfigFromServicesArray($providerName);
 
-        $this->additionalConfigKeys = $additionalConfigKeys = array_unique($additionalConfigKeys + ['guzzle']);
+        $this->additionalConfigKeys = $additionalConfigKeys = array_unique(array_merge($additionalConfigKeys, ['guzzle']));
```
Comment (AndyWendt) [inline src/Helpers/ConfigRetriever.php:45]: Can you write a corresponding failing test please?  Thank you for finding this!

**Comment by atymic** [comment]
ping @FI-LIFE

**Comment by FI-LIFE** [comment]
@atymic @AndyWendt sorry, i don't have a lot of experience for tests. See last comit

**Comment by Bobakanoosh** [comment]
Can this please be merged? Issue has been open for over a year and this is a one-line fix.

**Pull request closed without merging**